### PR TITLE
Improve Project Setup Script

### PR DESCRIPTION
**Pull request by seanpdoyle**
- Lock Ruby version to `2.2.2`
- Add `config/database.yml` and remove it from `.gitignore`. Instead of
  omitting (potentially sensitive) database configuration from source
  control, include the file and depend on environment variables for
  sensitive configuration values.
- Include dummy sensitive/secret keys for Devise and Rails.

What this will require in production:
- Define `SECRET_KEY_BASE` via `rake secret`
- Define `DEVISE_SECRET_KEY` via `rake secret`
- OPTIONALLY Define `DB_POOL`
- OPTIONALLY Define `DB_REAPING_FREQUENCY`

### CODE REVIEW

File: config/secrets.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+default: &default
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+
+  development:
+      <<: *default
+
```
Comment (jsom) [inline config/secrets.rb:6]: Is this indentation correct? Would have thought it should be outside of `default` key.

File: config/secrets.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+default: &default
```
Comment (jsom) [inline config/secrets.rb:1]: Should this be `.yml`

File: config/secrets.rb (hunk 1 above)
Comment (seanpdoyle) [inline config/secrets.rb:6]: No it's not!

File: config/secrets.rb (hunk 2 above)
Comment (seanpdoyle) [inline config/secrets.rb:1]: Fixed in https://github.com/codefordenver/fresh-food-connect-api/pull/45/commits/bd95cc0c9f14f8625eabdace8638de1b51aca45e

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+2.2.2
```
Comment (jsom) [inline .ruby-version:1]: The notes for this PR say lock it to 2.3.0 but here you have 2.2.2

**Pull request closed without merging**